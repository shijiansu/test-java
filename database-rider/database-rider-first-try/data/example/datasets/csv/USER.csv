ID, NAME
"1","@realpestano"
"2","@dbunit"
