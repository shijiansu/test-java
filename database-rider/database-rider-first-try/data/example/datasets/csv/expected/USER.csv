ID, NAME
"1","@dbrider"
"2","@dbunit"
